### PR TITLE
Fix user stat view

### PR DESCRIPTION
**Pull request by godwinpang**

### CODE REVIEW

File: src/entities/InducteePointsView.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+// later do some grouping by quarter
+@ViewEntity({
+  expression: (connection: Connection) =>
+    connection
+      .createQueryBuilder()
+      .select('appUser.id', 'user')
+      .addSelect('SUM(attendance.duration)', 'points')
+      .addSelect(
+        "SUM(CASE WHEN event.type = 'professional' THEN 1 ELSE 0 END)::int::bool",
```
Comment (thaigillespie) [inline src/entities/InducteePointsView.ts:14]: Is this a type cast to int and then bool?

File: src/entities/MemberPointsView.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { ViewEntity, Connection, ViewColumn } from 'typeorm';
+import { AppUser } from './AppUser';
+import { Attendance } from './Attendance';
+
+// later do some grouping by quarter
+// honestly we could maybe make this a query
+// idk we'll figure it out someday
+// - Godwin Sept 12th 2021 :sadge:
```
Comment (thaigillespie) [inline src/entities/MemberPointsView.ts:8]: 1 -> 0 



/s

File: src/migrations/1600027801558-FixPointsViews.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    await queryRunner.query(`DROP VIEW "member_points_view"`);
+    await queryRunner.query(
+      `CREATE VIEW "inductee_stat" AS SELECT "appUser"."id" AS "user", "appUser"."inductionClassQuarter" AS "inductionClass", SUM("attendance"."duration") AS "inductionPoints", COUNT("event"."type" = 'professional') > 0 AS "professional", COUNT("event"."type" = 'mentorship') > 0 AS "mentorship" FROM "app_user" "appUser" INNER JOIN "attendance" "attendance" ON "appUser"."id" = "attendance"."attendeeId"  INNER JOIN "event" "event" ON "event"."id" = "attendance"."eventId" WHERE "attendance"."isInductee" = true GROUP BY "appUser"."id", "appUser"."inductionClassQuarter"`
+    );
+    await queryRunner.query(
+      `INSERT INTO "typeorm_metadata"("type", "schema", "name", "value") VALUES ($1, $2, $3, $4)`,
+      [
+        'VIEW',
+        'public',
+        'inductee_stat',
```
Comment (thaigillespie) [inline src/migrations/1600027801558-FixPointsViews.ts:57]: I'm just wondering why inductee_stat is still here

**Review by thaigillespie — APPROVED** [review]
Lgtm, just had some questions

File: src/entities/InducteePointsView.ts (hunk 1 above)
Comment (godwinpang) [reply src/entities/InducteePointsView.ts:14]: yes - we can clean it up with exists later

**Pull request merged**